net/mwan3/files/usr/sbin/mwan3

```diff
@@ -81,8 +81,16 @@ ifup()
 	if [ "$up" = "1" ] \
 		&& [ -n "$l3_device" ] \
 		&& [ "$enabled" = "1" ]; then
-		ACTION=ifup INTERFACE=$1 DEVICE=$l3_device /sbin/hotplug-call iface
+		MWAN3_STARTUP=$MWAN3_STARTUP ACTION=ifup INTERFACE=$1 DEVICE=$l3_device sh /etc/hotplug.d/iface/15-mwan3
+		MWAN3_STARTUP=$MWAN3_STARTUP ACTION=ifup INTERFACE=$1 DEVICE=$l3_device sh /etc/hotplug.d/iface/16-mwan3-user
+	fi &
```

Comment:
Only this `&`  I think it's weird.
I know you want to put this in the background.
But it's easy to miss!
Does we have for this another solution?
Move this in a function?
mwan3_startup &

Reply:
Yes, that would be much clearer. I'll work on that change this evening.